aircan/dags/ng_data_dictionary_reporting.py

```diff
@@ -0,0 +1,148 @@
+import logging
+import datetime
+import time
+from pprint import pprint
+
+from airflow import DAG
+from airflow.exceptions import AirflowException
+from airflow.models import Variable
+import requests
+from urllib.parse import urljoin
+from airflow.operators.python_operator import PythonOperator
+from airflow.utils.dates import days_ago
+from airflow.operators.email_operator import EmailOperator
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+from string import Template
+from time import sleep
+
+CKAN_URL =  Variable.get("NG_CKAN_URL")
+SENDGRID_KEY =  Variable.get("NG_SENDGRID_API_KEY")
+MAIL_FROM = Variable.get("NG_MAIL_FROM")
+MAIL_TO = Variable.get("NG_MAIL_TO")
+
+args = {
+    'start_date': days_ago(0)
+}
+
+dag = DAG(
+    dag_id='ng_data_example_logs',
+    default_args=args,
+    schedule_interval='0 0 * * *'
```

Comment:
@sagargg we want it to run on weekly basis, e.g., at 00:00 on Sunday: `0 0 * * 0`

Reply:
done